go.mod

```diff
@@ -2,4 +2,4 @@ module github/tlaplus/foundation
 
 go 1.22.1
 
-require github.com/McShelby/hugo-theme-relearn v0.0.0-20241118130711-6cb54ccb05c2 // indirect
+require github.com/McShelby/hugo-theme-relearn v0.0.0-20250710213429-022fe7977283 // indirect
```

Comment:
Why does the July newsletter require an update of the hugo theme?

Reply:
From summary:

>Had to update some stuff so the site builds in newer versions of Hugo (first commit). For whatever reason the Hugo devs have chosen to keep the site in perma-0.x versioning mode and continually break backward compatibility.

So the options were to install an old version of Hugo so that I could build the site as-is (difficult to do in Arch Linux and also Homebrew on macOS) or just update the Hugo theme. Before this I was just switching to NixOS where I could pin the version of Hugo used but I had to write this on my laptop (macbook) while on the road.